plugins/napari_ADS/_widget.py

```diff
@@ -161,6 +161,16 @@ def __init__(self, napari_viewer):
         )
 
         self.available_models = ads_utils.get_existing_models_list()
+
+        if self.available_models == None:
+            if self.show_ok_cancel_message('First time opening pluging - models must be downloaded'):

```

Comment:
Pluging -> plugin?

Reply:
Also, maybe the following description better illuminates the issue. Since, technically the model folder could get deleted, but it wouldn't be the first time opening the plugin, so the user might be confused...

```suggestion
            if self.show_ok_cancel_message('Model folder is missing. (Hint: Is this the first time opening the plugin?)\n\nPress 'OK' to download the required models.'):

```